docs/source/pre_application/running_a_node.rst

```diff
@@ -230,6 +231,14 @@ This step starts the PRE node.
     nucypher/nucypher:latest          \
     nucypher ursula run
 
+
+.. note::
+
+    Setting the ``--restart`` parameter to ``unless-stopped`` ensures that the Docker
+    container will be automatically restarted if it exited, except if the container was
```

Comment:
To make this a bit more useful for a less technical audience, perhaps we can say "crashed" instead of "exited" (or something similar)?

Reply:
Technically it does restart even if there was a valid exit (exit code of 0). Thinking more about this, perhaps the restart policy should be `on-failure` instead? 🤔  (cc @vepkenez )

I'm thinking about the case where a node is proactively shut down because it is no longer bonded to a staking provider.

Reply:
> Technically it does restart even if there was a valid exit (exit code of 0). Thinking more about this, perhaps the restart policy should be `on-failure` instead? 🤔 (cc @vepkenez )
> 
> I'm thinking about the case where a node is proactively shut down because it is no longer bonded to a staking provider.

Like for example if we have a thing where we check the blockchain and kill the process if we are no longer bonded?
Yeah... maybe.  Or we could just handle that in the rest middleware and return 400s. Not sure.  It seems strange to just kill the process in that case.

Reply:
We already have thing 😅  - https://github.com/nucypher/nucypher/pull/2886/commits/a8aa869c2d41a73f2fc11b8020e5655f0c1ad6d3.

Reply:
^ @KPrasch / @vepkenez

Reply:
Went with `on-failure` instead of `unless-stopped`. @vepkenez something for you to think about with respect to `nucypher-ops`.